extensions/sql-database-projects/README.md

```diff
@@ -9,8 +9,9 @@ Please report issues and feature requests [here.](https://github.com/microsoft/a
 
 ## Getting Started with Database Projects
 
-* Create a new database project by selecting File -> New Database Project, going to the Database Projects viewlet under Explorer, or by searching for **New Database Project** in the command palette (Ctrl + Shift + P).
-* Existing database projects can be opened from File -> Open File, or via **Open Database Project** in the command palette.
+* Create a new database project by going to the **Projects** viewlet under Explorer, or by searching for **New Database Project** in the command palette.
+* Existing database projects can be opened via **Open Database Project** in the command palette.
```

Comment:
"in the command palette" vs "from the command palette", you are using both here, i don't know which one is more proper 😃

Reply:
They're all good!  Just changing it up so it doesn't sound robotic